apps/frontend/src/modules/projects/libs/constants/contribution-analytics-date-max-range.constant.ts

```diff
@@ -0,0 +1,3 @@
+const CONTRIBUTION_ANALYTICS_DATE_MAX_RANGE = 30;
```

Comment:
this should stay as ANALYTICS_DATE_MAX_RANGE, and for another constant - it's not a max range, it's just an initial range, so let's rename it to ANALYTICS_DEFAULT_DATE_RANGE. So the max range should stay the same - 30 days, but the default preselected value should be 7 days
```suggestion
const ANALYTICS_DATE_MAX_RANGE = 30;
```